budget/test/test_duplicate_budget.yml

```diff
@@ -4,17 +4,43 @@
   !context
     uid: 'res_users_beth'
 -
+  I first create a budget item
+-
+  !record {model: budget.item, id: budget_item_1}:
```

Comment:
If it's not there already, please add a comment with reference to the bug nearby.